code/lib/ui/src/settings/about.tsx

```diff
@@ -45,12 +46,15 @@ const Subheader = styled.div({
 const UpdateMessage = styled.div<{ status: 'positive' | 'negative' | string }>(
   ({ status, theme }) => {
     if (status === 'positive') {
-      return { background: theme.background.positive, color: theme.color.positive };
+      return { background: theme.background.positive, color: theme.color.positiveText };
     }
     if (status === 'negative') {
-      return { background: theme.background.negative, color: theme.color.negative };
+      return { background: theme.background.negative, color: theme.color.negativeText };
     }
-    return { background: '#EAF3FC', color: theme.color.darkest };
+    return {
+      background: theme.base === 'light' ? '#EAF3FC' : theme.color.darkest,
```

Comment:
#EAF3FC seems like a color used in a few places - is it worth adding to the theme?

Reply:
I don't think so. It's only used in two places (and soon only one). For now I think it's okay to be a one-off.